### PR TITLE
Broken links audit #1903

### LINKED ISSUE #1903: [Edit] Broken Links Audit: Entries 1-10
### Type of Edit (select all that apply)

Fix grammar, spelling, formatting

### Description (optional)

As part of Docs maintenance we have conducted an audit of our directories and compiled a number of entries with broken links. This issue puts forward the first ten for updating. For each entry the link highlighted below must be corrected to the new (or an alternative) path for the relevant resource.

| # | location | link |
| -- | -- | -- |
| 0 | docs/documentation/style-guide.md | https://raw.githubusercontent.com/Codecademy/docs/main/media/ |
| 1 | docs/content/docs-community/concepts/contributions/terms/docs-repository/docs-repository.md | https://www.codecademy.com/resources/docs/docs-community/contributing-to-docs/reviews |
| 2 | docs/content/docs-community/concepts/contributions/terms/reviews/reviews.md | https://www.codecademy.com/resources/docs/docs-community/docs-repository |
| 3 | docs/content/docs-community/concepts/contributions/contributions.md | https://www.codecademy.com/resources/docs/general/docs-community/reviews |
| 4 | docs/content/go/concepts/composition/composition.md | https://www.codecademy.com/resources/docs/go/structs |
| 5 | docs/content/go/go.md | https://www.codecademy.com/resources/docs/general/cloud-computing |
| - | docs/content/go/go.md | https://www.codecademy.com/resources/docs/general/artificial-intelligence |
| 6 | docs/content/open-source/concepts/open-source-software/open-source-software.md | https://www.codecademy.com/resources/docs/open-source/vim |
| 7 | docs/content/open-source/concepts/node-js/node-js.md | https://www.codecademy.com/resources/docs/general/terminal |
| 8 | docs/content/open-source/concepts/font-awesome/font-awesome.md | https://www.codecademy.com/resources/docs/general/bootstrap |
| - | docs/content/open-source/concepts/font-awesome/font-awesome.md | https://kit.fontawesome.com/kitcode.js |
| 9 | docs/content/python/concepts/built-in-functions/terms/any/any.md | https://www.codecademy.com/resources/docs/python/list |

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by JohnSteeleCG**
### Description

Made updates to broken links in the following documents: 
* docs/content/docs-community/concepts/contributions/terms/docs-repository/docs-repository.md
* docs/content/docs-community/concepts/contributions/terms/reviews/reviews.md
* docs/content/docs-community/concepts/contributions/contributions.md
* docs/content/go/concepts/composition/composition.md
* docs/content/go/go.md
* docs/content/open-source/concepts/open-source-software/open-source-software.md
* docs/content/open-source/concepts/node-js/node-js.md
* docs/content/open-source/concepts/font-awesome/font-awesome.md
* docs/content/python/concepts/built-in-functions/terms/any/any.md

Did not make update to https://raw.githubusercontent.com/Codecademy/docs/main/media/ or https://kit.fontawesome.com/kitcode.js as they seem intentional. Closes #1903.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by caupolicandiaz — APPROVED** [review]
@JohnSteeleCG Looks good.

**Review by SSwiniarski — APPROVED** [review]
@JohnSteeleCG Looks good

File: content/open-source/concepts/open-source-software/open-source-software.md [hunk 1]
```diff
@@ -46,5 +46,5 @@ Some examples of OSS software (along with their license and release year) includ
...
 |              [React](https://www.codecademy.com/resources/docs/react)              |               MIT                |                2013                | A [front-end](https://www.codecademy.com/resources/docs/general/front-end) library, written in JS, that is component-based and used for building web user interfaces.                                                                        |
 |    [React Native](https://www.codecademy.com/resources/docs/react/react-native)    |               MIT                |                2015                | A mobile development framework that works across different operating systems like iOS and Android.                                                                                                                                           |
-|          [Vim](https://www.codecademy.com/resources/docs/open-source/vim)          |               Vim                |                1991                |
+|          [Vim](https://www.codecademy.com/resources/docs/command-line/vim)          |               Vim                |                1991                |
```
Comment (SSwiniarski) [inline content/open-source/concepts/open-source-software/open-source-software.md:49]:
```suggestion
|          [Vim](https://www.codecademy.com/resources/docs/command-line/vim)         |               Vim                |                1991                |
```

**Review by SSwiniarski — APPROVED** [review]

**Comment by yangc95** [comment]
Thanks a bunch @JohnSteeleCG links are fixed here: Docs Community [Contributions](https://www.codecademy.com/resources/docs/docs-community/contributions): [Docs Repository](https://www.codecademy.com/resources/docs/docs-community/contributions/docs-repository), [Reviews](https://www.codecademy.com/resources/docs/docs-community/contributions/reviews), [Go](https://www.codecademy.com/resources/docs/go): [Composition](https://www.codecademy.com/resources/docs/go/composition), Open Source: [Font Awesome](https://www.codecademy.com/resources/docs/open-source/font-awesome), [Node.js](https://www.codecademy.com/resources/docs/open-source/node-js), [Open Source Software](https://www.codecademy.com/resources/docs/open-source/open-source-software), Python Built-in-Functions: [any()](https://www.codecademy.com/resources/docs/python/built-in-functions/any)

cc: @caupolicandiaz @SSwiniarski @KTom101

**Pull request merged**